test/e2e/storage/csi_volumes.go

```diff
@@ -320,9 +227,18 @@ var _ = utils.SIGDescribe("CSI Volumes", func() {
 		}
 		for _, t := range tests {
 			test := t
+			f := framework.NewDefaultFramework("csiworkload")
+
 			It(test.name, func() {
+				cs := f.ClientSet
+				csics := f.CSIClientSet
+				ns := f.Namespace
+				driver := drivers.InitMockCSIDriver()
```

Comment:
why do we Init a mock driver specifically here, then do a create CSI driver later?
Shouldn't the mock driver be one of many `csiDriver`'s permutations that we are programatically testing?

Reply:
`InitMockCSIDriver` doesn't initialize the driver. It just creates the TestDriver struct for it.

I don't know why the mock is treated special here.